state/integration_test.go

```diff
@@ -110,3 +116,32 @@ func (s *IntegrationTestSuite) TestGetBalance() {
 		require.Equal(&expectedBal, bal)
 	}
 }
+
+// This test can be used to generate a json encoded block for other test data,
+// such as that in share/availability/light/testdata
+func (s *IntegrationTestSuite) TestGenerateJSONBlock() {
```

Comment:
can delete, but this will just generate the sample block for us should we ever need to again